lib/resolvers/aix/networking_resolver.rb

```diff
@@ -15,10 +15,13 @@ def post_resolve(fact_name)
           end
 
           def read_netstat(fact_name)
+            @fact_list[:interfaces] = {}
             output = Facter::Core::Execution.execute('netstat -rn', logger: log)
             output = output.split("\n").select { |line| (line =~ /\s\s[0-9]+.[0-9]+.[0-9]+.[0-9]+|\s\s.*:[0-9a-f]+/) }
```

Comment:
`split("\n")` can be replaced with `each_line`

Reply:
Done